cmd/ci_run.go

```diff
@@ -143,12 +135,14 @@ func parseCIVariables(vars []string) (map[string]string, error) {
 
 func init() {
 	ciCreateCmd.Flags().StringP("project", "p", "", "project to create pipeline on")
+	ciCreateCmd.Flags().MarkDeprecated("project", "project is inferred from branch only")
```

Comment:
So despite the strangeness with `"project"` being pulled in by `"token"`, the `--project` flag is still relevant in `getCIRunOptions` in this file, and I think it actually behaves correctly there, we have tests for that one. It's also an important feature that folks can trigger ci pipelines in other projects, as lab is used in combination with the `lab ci status --wait` command to orchestrate downstream pipelines (unsure how many people do this, but that's the work flow we built for)

Reply:
The workflow itself and ideas there stem from https://github.com/zaquestion/lab/issues/240

Reply:
I'll take a closer look at this then. Because I've implemented the downstream pipeline in other commands (like `lab ci artifacts`) as a builtin feature, picking the downstream project/project through the current "upstream" pipeline information (API response). Letting the user to specify any project, regardless it being the downstream pipeline or not, is kinda weird considering how the remaining of the code and commands behaves.

I understand how useful it can be, I just think it's odd enough to break the consistency between commands. One thing I propose is: make this "--project" option strict to downstream pipelines by adding another option like done in `ci_artifacts` with `--follow`. What do you think?